pyfolio/perf_attrib.py

```diff
@@ -164,46 +169,63 @@ def show_perf_attrib_stats(returns, positions, factor_returns,
         pos_in_dollars=pos_in_dollars,
     )
 
-    perf_attrib_stats = create_perf_attrib_stats(perf_attrib_data)
+    perf_attrib_stats, risk_exposure_stats =\
+        create_perf_attrib_stats(perf_attrib_data, risk_exposures)
```

Comment:
Thoughts on having `perf_attrib()` and `create_perf_attrib_stats()` returning things in the same order for consistency? I.e. changing this to `risk_exposure_stat, perf_attrib_stats = ...`